src/backend/app/Models/Token.php

```diff
@@ -4,11 +4,50 @@
 
 use App\Models\Base\BaseModel;
 use App\Models\Transaction\Transaction;
+use Illuminate\Database\Eloquent\Builder;
 use Illuminate\Database\Eloquent\Relations\BelongsTo;
 use Illuminate\Database\Eloquent\Relations\MorphOne;
+use Illuminate\Support\Facades\DB;
 
+/**
+ * Generic Token Model for all types of tokens i.e meter token, shs token, etc.
+ *
+ * @property string $token
+ * @property float  $load
+ * @property int    $transaction_id
+ * @property string $token_type
+ * @property int    $device_id
+ * @property int    $token_amount
```

Comment:
`load` and `token_amount` seem redundant.

I like what you had suggested in your previous PR to have the following fields

- `token_type` (can be `time` or `energy`)
- `token_value` (can be `kWH`, `day`, `month`) Although this name is not fully intuitive. Maybe something like `token_unit`?
- `token_amount` (can be the actual amount) TBC if this should better be a float to enable for example 2.5 kWH tokens?

Reply:
Alright, I will work on it. Thanks

Reply:
How do you suggest getting rid of an already existing `load` table field? I didn't want to potential break existing company table.

Reply:
Can't we just rename it in the migration? Then, of course, we would have to check where in the code `MeterToken->load` is used, and replace it with the new field name.

But maybe I'm thinking to easy 😅

Reply:
😄 easy/simple solutions are always the best. I will go with this; I was overthinking it 🙃